src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -3175,6 +3199,31 @@ class spell_pri_vampiric_touch : public AuraScript
     }
 };
 
+// 391286 - Whispering Shadows (Dot Application)
+// Triggered by 205385 - Shadow Crash
+class spell_pri_whispering_shadows_dummy : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_PRIEST_WHISPERING_SHADOWS, SPELL_PRIEST_VAMPIRIC_TOUCH });
+    }
+
+    bool Load() override
+    {
+        return GetCaster()->HasAura(SPELL_PRIEST_WHISPERING_SHADOWS);
+    }
+
+    void HandleEffectHitTarget(SpellEffIndex /*effIndex*/) const
+    {
+        GetCaster()->CastSpell(GetHitUnit(), SPELL_PRIEST_VAMPIRIC_TOUCH, TRIGGERED_FULL_MASK);
```

Comment:
it is often preferred to use the necessary trigger flags rather than setting them all, it's better for debugging later if bugs arise for whatever reason or even if several spells are linked in some way